exercises/en/test_04_07.py

```diff
@@ -0,0 +1,38 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    # If an assertion fails, the message will be displayed
+
+    assert not pokemon_circleplot is None, "Your answer does not exist. Have you passed in the correct variable?"
+    assert type(pokemon_circleplot) == type(alt.Chart()), "Your answer is not an Altair Chart object. Check to make sure that you have assigned an alt.Chart object to pokemon_circleplot."
+    assert pokemon_circleplot.mark == 'circle', "Make sure you are using the 'mark_circle' to generate a circleplot."
+
+    assert (
+        pokemon_circleplot.encoding.x.shorthand in {'gen:ordinal', 'gen:O'} or 
```

Comment:
What about just `gen` does that not work?

Reply:
Only having `gen` makes altair infer the type as `'quantitative'` instead, but `gen` with `type='ordinal'` works. I modified the test to accept it.
Thanks!